File: dayTwo/javaScript/After Break/averageScoresGrades.js

function average_grade_of(firstScore,secondScore,thirdScore){
    let totalScore = firstScore + secondScore + thirdScore;
    let average = totalScore / 3
    
    if (average >= 90 && average <= 100) {
        console.log("\'A\'")
    }
    else if(average >= 80 && average < 90) {
        console.log("\'B\'")
    }
    else if(average >= 70 && average < 80) {
        console.log("\'C\'")
    }
    else if(average >= 60 && average < 70) {
        console.log("\'D\'")
    }  
    else {
        console.log("\'F\'")
    }
           
}

average_grade_of(80, 70, 60)

File: dayTwo/python/After Break/average_scores_grades.py

def averageGradeOf(first_score, second_score,third_score):
    total_scores = first_score + second_score + third_score
    average = total_scores / 3
    
    if 90 <= average <= 100 :
        print("\'A\'")
    elif 80 <= average < 90 :
        print("\'B\'")
    elif 70 <= average < 80 :
        print("\'C\'")
    elif 60 <= average < 70 :
        print("\'D\'")
    else :
        print("\'F\'")
        

averageGradeOf(50,70,40)        

